docs/cli/fleet-cli/fleet_apply.md

```diff
@@ -46,6 +46,11 @@ fleet apply [flags] BUNDLE_NAME PATH...
       --target-namespace string                Ensure this bundle goes to this target namespace
       --targets-file string                    Addition source of targets and restrictions to be append
       --username string                        Basic auth username for helm repo
+      --driven-scan                            Use driven scan. Bundles are defined by the user.
+                                               Paths will be defined by the tuple <base, options> separated by a separator.
+                                               The default separator is the character :. If you need to use a different one, 
```

Comment:
nit: I'd quote the separator, just to make it really clear that this is not a typo with the separator having been omitted and `:` being part of the sentence, as in "here's the separator : <blah>".
Something like this: (or with backticks, or double quotes, whatever)
```suggestion
                                               The default separator is the character ':'. If you need to use a different one, 
```

Reply:
changed as suggested